content/en/docs/reference/access-authn-authz/validating-admission-policy.md

```diff
@@ -76,12 +76,11 @@ kind: ValidatingAdmissionPolicyBinding
 metadata:
   name: "demo-binding-test.example.com"
 spec:
-  policy: "replicalimit-policy.example.com"
+  policyName: "demo-policy.example.com"
   matchResources:
-    namespaceSelectors:
-    - key: environment,
-      operator: In,
-      values: ["test"]
+    namespaceSelector:
+      matchLabels:
+        environment: test
```

Comment:
Issues like this would be easy to detect if we make the manifest an example YAML.
For example:

1. move the YAML snippet into a separate file `content/en/examples/policy/validating-admission-ex.yaml` and,
2. reference it as `{{< codenew file="policy/validating-admission-ex.yaml" >}}` in the markdown page,
3. (optionally) amend `content/en/examples/examples_test.go` so that it can be validated against current and future k8s releases.

Reply:
I can do that in a follow up as part of improving this doc: https://github.com/kubernetes/website/issues/38504

I've added your comment to that issue. The focus of this PR is to solely fix the incorrect policy asap since I struggled quite a bit when going through the docs with the incorrect examples. I had to look up the golang API docs to figure out what the spec was.

Reply:
Fair enough.
The existing examples are all tested against the current release, by the way, although we may want to make this test automated.